curriculum/challenges/espanol/06-quality-assurance/quality-assurance-and-testing-with-chai/run-functional-tests-on-an-api-response-using-chai-http-iii---put-method.md

```diff
@@ -1,57 +1,65 @@
 ---
 id: 587d824f367417b2b2512c5a
-title: Run Functional Tests on an API Response using Chai-HTTP III - PUT method
+title: Ejecutar pruebas funcionales en la respuesta de un API usando Chai-HTTP III - método PUT
 challengeType: 2
 forumTopicId: 301590
 dashedName: run-functional-tests-on-an-api-response-using-chai-http-iii---put-method
 ---
 
 # --description--
 
-As a reminder, this project is being built upon the following starter project on [Replit](https://replit.com/github/freeCodeCamp/boilerplate-mochachai), or cloned from [GitHub](https://github.com/freeCodeCamp/boilerplate-mochachai/).
+Como recordatorio, este proyecto está siendo construido con base en el siguiente proyecto inicial [Repl.it](https://replit.com/github/freeCodeCamp/boilerplate-mochachai), o clonado desde [GitHub](https://github.com/freeCodeCamp/boilerplate-mochachai/).
```

Comment:
Not a big deal, but I am not sure why this has changed to `Repl.it`. A while ago (when Repl.it became Replit), we changed all of these links to match.

Reply:
@nhcarrigan

Reply:
Presumably because it's in the translation memory.

Reply:
@ojeytonwilliams @ShaunSHamilton let us know if you want to keep the discussion here or resolve this and move it to a dedicated thread or chat.